include/mgp.hpp

```diff
@@ -2710,9 +2716,9 @@ inline void Node::RemoveLabel(const std::string_view label) {
   mgp::vertex_remove_label(this->ptr_, mgp_label{.name = label.data()});
 }
 
-inline std::map<std::string, Value> Node::Properties() const {
+inline std::unordered_map<std::string, Value> Node::Properties() const {
```

Comment:
This is a bit of a breaking change for current procedures that are using std::map in MAGE or other procedures. Can we have two different functions? What was the reason to modify this so it returns unordered_map? I know it is faster but is there anything else?

Reply:
The reason is it is just faster.